fighters/common/src/general_statuses/mod.rs

```diff
@@ -90,7 +90,7 @@ pub unsafe fn status_pre_DamageAir(fighter: &mut L2CFighterCommon) -> L2CValue {
 
 #[skyline::hook(replace = smash::lua2cpp::L2CFighterCommon_sub_DamageFlyCommon_init)]
 pub unsafe fn damage_fly_common_init(fighter: &mut L2CFighterCommon) {
-    ControlModule::set_command_life_extend(fighter.module_accessor, 5);
+    ControlModule::set_command_life_extend(fighter.module_accessor, 0);
```

Comment:
this line shouldn't even exist if you wanted to get rid of this

Reply:
u get a comment out, take it or leave it

Reply:
lgtm